src/main/controllers/system-admin/BlobViewSubscriptionResubmitConfirmationController.ts

```diff
@@ -0,0 +1,33 @@
+import { PipRequest } from '../../models/request/PipRequest';
+import { Response } from 'express';
+import { cloneDeep } from 'lodash';
+import { PublicationService } from '../../service/PublicationService';
+import { LocationService } from '../../service/LocationService';
+import { SubscriptionService } from '../../service/SubscriptionService';
+
+const publicationService = new PublicationService();
+const locationService = new LocationService();
+const subscriptionService = new SubscriptionService();
+
+export default class BlobViewSubscriptionResubmitConfirmationController {
+    public async get(req: PipRequest, res: Response): Promise<void> {
```

Comment:
Similar comment here - we'll need a check to make sure artefact ID is provided

Reply:
Updated